apps/test-suite/tests/NewNotifications.js

```diff
@@ -943,9 +943,61 @@ export async function test(t) {
           },
           16000
         );
+
+        t.it(
+          'triggers a repeating daily notification. only first event is verified.',
+          async () => {
+            let timesSpyHasBeenCalled = 0;
+            const subscription = Notifications.addNotificationReceivedListener(() => {
+              timesSpyHasBeenCalled += 1;
+            });
+            const dateNow = new Date();
+            await Notifications.scheduleNotificationAsync({
+              identifier,
+              content: notification,
+              trigger: {
+                hour: dateNow.getHours(),
+                minute: dateNow.getMinutes() + 1,
+              },
+            });
+            await waitFor(60000);
+            t.expect(timesSpyHasBeenCalled).toBe(1);
+            subscription.remove();
+          },
+          70000
+        );
       }
 
       if (Platform.OS === 'ios') {
+        t.it(
```

Comment:
There is an iOS and Android version of this test because Android uses 

  `addNotificationReceivedListener `

and iOS uses 

   `setNotificationHandler ` 

to verify that the notification is triggered. If I recall correctly, iOS didn't work with `addNotificationReceivedListener`.